ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 5.6.2f1
+m_EditorVersion: 5.6.0f3
```

Comment:
Couldn't figure out an easy automated way to accept them so I just did it manually - seems crazy that it's not an option.